docs/cn/DATA-FILES.md

```diff
@@ -10,14 +10,52 @@
 
 如果你仍然希望使用 Hexo 2.x，你依旧可以按老的方式进行配置。NexT 仍然兼容 Hexo 2.x（但可能会出现错误）。
 
-## 带来的好处
+<h2 align="center">选择 1：Hexo 方式</h2>
 
-使用这一特性，你现在可以将你的全部配置置于同一位置（`source/_data/next.yml`），并且不需要修改 `next/_config.yml`。如果在新的 release 中出现了任何新的选项，那么你只需要从 `next/_config.yml` 中将他们复制到 `source/_data/next.yml` 中并设置它们的值为你想要的选项。
+使用这一方式，你的全部配置都将置于 hexo 主要配置文件中（`hexo/_config.yml`），并且不需要修改 `next/_config.yml`，或者创建什么其他的文件。但是所有的主题选项必须放置在 `theme_config` 后。
```

Comment:
These words in the original `DATA-FILES.md` not translated:

```
But you must preserve double spaces indents within theme_config option.
```
Especially the **double spaces** not translated.

Reply:
Actually yeah, I didn't thought it might be make some misunderstanding.